src/main/java/com/synopsys/integration/jira/common/server/configuration/JiraServerBasicAuthRestConfigBuilder.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * int-jira-common
+ *
+ * Copyright (c) 2023 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.jira.common.server.configuration;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.synopsys.integration.builder.BuilderPropertyKey;
+import com.synopsys.integration.builder.BuilderStatus;
+
+public class JiraServerBasicAuthRestConfigBuilder extends JiraServerRestConfigBuilder<JiraServerBasicAuthRestConfigBuilder, JiraServerBasicAuthRestConfig> {
+    public static final BuilderPropertyKey AUTH_USERNAME = new BuilderPropertyKey("JIRA_AUTH_USERNAME");
+    public static final BuilderPropertyKey AUTH_PASSWORD = new BuilderPropertyKey("JIRA_AUTH_PASSWORD");
+
+    @Override
+    protected JiraServerBasicAuthRestConfigBuilder getThis() {
+        return this;
+    }
+
+    @Override
+    public Set<BuilderPropertyKey> getAuthenticationProperties() {
+        Set<BuilderPropertyKey> propertyKeys = new HashSet<>();
+        propertyKeys.add(AUTH_USERNAME);
+        propertyKeys.add(AUTH_PASSWORD);
+        return propertyKeys;
+    }
+
+    @Override
+    protected JiraServerBasicAuthRestConfig buildWithoutValidation() {
+        URL jiraUrl = null;
+        try {
+            jiraUrl = new URL(getUrl());
+        } catch (MalformedURLException e) {
```

Comment:
Just a warning from IntelliJ. Since nothing is done with the catch, rename the variable to ignored.